certified-connectors/E-Sign/apiDefinition.swagger.json

```diff
@@ -2,12 +2,12 @@
   "swagger": "2.0",
   "info": {
     "title": "E-Sign",
-    "description": "E-Sign Anything. Anywhere. It’s easy to e-sign and send your documents on any device with our secure electronic signature system.",
-    "version": "3.1",
+    "description": "E-Sign Anything. Anywhere. It\u2019s easy to e-sign and send your documents on any device with our secure electronic signature system.",
+    "version": "4.0",
```

Comment:
@eSignHQ are you sure that this version is correct? I think the version should be 5.0 for the update.